main.cpp

```diff
@@ -125,419 +152,409 @@ struct CarWash
-    //stores the amount of water used per week.     
-    float waterUsedPerWeek = 200.f;            
-    //stores amount of profit made per week         
-    float profitPerWeek = 495.95f;               
-    //number of cars serviced per day               
+
+    int numVacuumCleaners = 3; 
+    int numEcoFriendlyCleaningSupplies = 20;       
+    float waterUsedPerWeek = 200.f;                    
+    float profitPerWeek = 495.95f;                         
     int numberOfCarsServiced = 10;               
     
-    struct Car //5)                                 Note that the nested type 'Car' is related to the 'CarWash' 
+    struct Car                                 
     {
-        //2) member variables with relevant data types.  the names are appropriate for the U.D.T.'s purpose.
         bool isAPickupTruck = false;
         float gasMileage = 26.2f;        
         int year = 1985;
         std::string manufacturer = "Toyota";
         std::string model = "Corolla";
 
-        //3) a member function whose parameter has a default value.
-        //the parameter name is related to the work the function will perform.
+   
         void fillTank(double fuelAmountInGallons = 2.0);  
         void breakDown(std::string failureType, bool requiresTow = false);
         int getMilesTraveledAnnually(bool includeUberLyftTrips);
     };
 
-    //wash and wax car
-    void washAndWaxCar( Car car ); //4) a member function whose parameter is a UDT.
-    //charge customer
+   
+    void washAndWaxCar( Car car ); 
     float chargeCustomer(float discountPercentage);
-    //detail the car interior
     void detailInterior( Car car );
     
-    //5) a member variable whose type is a UDT.
     Car carBeingServiced;  
 };
 
 
-/*
-Thing 1) Octopus
-5 properties:
-    1) number of arms
-    2) amount of ink
-    3) color
-    4) shape
-    5) size
-3 things it can do:
-    1) catch a fish
-    2) change color
-    3) squirt ink
- */
+
 struct Octopus
 {
-    //number of arms
     int numOfArms;
-    //amount of ink
     float inkAmount;
-    //color
     char color;
-    // shape
     char shape;
-    //size in ft
     float size;
 
-    //catch a fish, bool to return fishCaught
-    bool catchAFish(int whichArm);
-    //change color
-    void changeColor(char color, char secondaryColor);
-    //squirt ink
+    bool catchAFish(int whichArm, int distanceToFish);
+    void changeColor(char newColor, char thisColor);
     void squirtInk(float inkSize, char escapeDirection);
 };
-/*
-Thing 2) Band
-5 properties:
-    1) Number of people in band
-    2) Type of band
-    3) Number of albums sold
-    4) Budget
-    5) Number of songs written
-3 things it can do:
-    1) Play for an audience
-    2) Record a song
-    3) Post on social media
- */
+
+bool Octopus::catchAFish(int whichArm, int distanceToFish)
+{
+    bool fishWasCaught = false;
+    if (whichArm + distanceToFish < 10)
+    {
+        fishWasCaught = true;
+    }
+
+    return fishWasCaught;
```

Comment:
you can just write 
`return (whichArm + distanceToFish < 10);`
the stuff between the `( )` will be evaluated and the result of that evaluation will be the value that is returned.
it's nice and concise.
- [ ] revise